modules/tinymce/src/core/test/ts/browser/fmt/ListIndentationFormatTest.ts

```diff
@@ -0,0 +1,133 @@
+import { Keys } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { PlatformDetection } from '@ephox/sand';
+import { Css, SugarElement } from '@ephox/sugar';
+import { TinyContentActions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.fmt.ListItemFormatTest', () => {
+  const platform = PlatformDetection.detect().browser;
+  const hook = TinyHooks.bddSetup<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+    plugins: 'lists'
+  }, [], true);
+
+  const checkCss = () => {
+    const item2 = hook.editor().dom.select('li.testClass')[0];
+
+    const sugarItem2 = SugarElement.fromDom(item2);
+
+    const color = Css.get(sugarItem2, 'color') === 'rgb(34, 47, 62)';
+    const backgroundColor = Css.get(sugarItem2, 'background-color') === 'rgba(0, 0, 0, 0)';
+    const fontStyle = Css.get(sugarItem2, 'font-style') === 'normal';
+    const fontWeight = Css.get(sugarItem2, 'font-weight') === '400';
+    const fontFamily = Css.get(sugarItem2, 'font-family') ===
+      (platform.isChromium() ?
+        '-apple-system, "system-ui", "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif' :
+        '-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif');
+
+    return color && backgroundColor && fontStyle && fontWeight && fontFamily;
+  };
+
+  context('Apply list formatting', () => {
```

Comment:
Context is also about clumping together different thing say a module has three functions A, B, C I tend to use context to wrap A, B, C and have multiple it's for each of those functions since then you can set only on A if you are working on fixing some bug in A and just run the tests for that. So if the context here is applying list formatting you can wrap that in a context and focus on that if you have more than one test for it.